### PR TITLE
Have OLM metadata in source repo for reference

### PR DESCRIPTION
**Pull request by aliok**
Fixes https://github.com/knative/eventing-operator/issues/68

### CODE REVIEW

**Comment by aliok** [comment]
cc @houshengbo @k4leung4

File: deploy/olm-catalog/knative-eventing-operator/0.11.0/knative-eventing-operator.v0.11.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    For complete Knative Eventing documentation, see
+    [Knative eventing](https://www.knative.dev/docs/eventing/) or
+    [Knative docs](https://www.knative.dev/docs/) to learn about Knative.
+  maturity: alpha
+  version: 0.11.0
+  minKubeVersion: ''
+  keywords:
+    - serverless
+    - eventing
+    - camel
```
Comment (matzew) [inline deploy/olm-catalog/knative-eventing-operator/0.11.0/knative-eventing-operator.v0.11.0.clusterserviceversion.yaml]: there is no camel, nor kafka

File: deploy/olm-catalog/knative-eventing-operator/0.11.0/eventing_v1alpha1_knativeeventing_crd.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  name: eventings.operator.knative.dev
+spec:
+  group: operator.knative.dev
+  names:
+    kind: Eventing
+    listKind: EventingList
+    plural: eventings
+    singular: eventing
+    shortNames:
+      - ke
```
Comment (matzew) [inline deploy/olm-catalog/knative-eventing-operator/0.11.0/eventing_v1alpha1_knativeeventing_crd.yaml]: do we really need this shortname ? /cc @cardil

File: deploy/olm-catalog/knative-eventing-operator/0.11.0/knative-eventing-operator.v0.11.0.clusterserviceversion.yaml (hunk 1 above)
Comment (aliok) [reply deploy/olm-catalog/knative-eventing-operator/0.11.0/knative-eventing-operator.v0.11.0.clusterserviceversion.yaml]: +1, let me get rid of those here and also in operatorHub

File: deploy/olm-catalog/knative-eventing-operator/0.11.0/eventing_v1alpha1_knativeeventing_crd.yaml (hunk 2 above)
Comment (aliok) [reply deploy/olm-catalog/knative-eventing-operator/0.11.0/eventing_v1alpha1_knativeeventing_crd.yaml]: hmm, I started this PR before the shortname discussion. What's the latest situation there ? :rofl:

Comment (cardil) [reply deploy/olm-catalog/knative-eventing-operator/0.11.0/eventing_v1alpha1_knativeeventing_crd.yaml]: Short names should be removed

Comment (aliok) [reply deploy/olm-catalog/knative-eventing-operator/0.11.0/eventing_v1alpha1_knativeeventing_crd.yaml]: hmm, hold on :)

this OLM metadata is for the 0.11 version of the operator. In that version, short name is still there: https://github.com/knative/eventing-operator/blob/v0.11.0/config/300-eventing-v1alpha1-knativeeventing-crd.yaml#L12

So, let's keep this one for consistency. The CRD in this PR is identical to https://github.com/knative/eventing-operator/blob/v0.11.0/config/300-eventing-v1alpha1-knativeeventing-crd.yaml#L12

**Comment by aliok** [comment]
Created https://github.com/operator-framework/community-operators/pull/1108 for fixing the metadata in OperatorHub

**Pull request merged**